docs/usage/cli.md

```diff
@@ -68,6 +68,14 @@ Compile the entire `src` directory and output it to the one concatenated file.
 $ babel src --out-file script-compiled.js
 ```
 
+### Ignore files
+
+Ignore spec and test files
+
+```sh
+$ babel src --out-dir lib --ignore spec.js,test.js
```

Comment:
I think a number of projects these days (including ours), include the .spec files adjacent to the file it is testing).

Happy to update to a different type of file to ignore?